src/test/README.md

```diff
@@ -53,6 +53,11 @@ The `-printtoconsole=1` after the two dashes redirects the debug log, which
 would normally go to a file in the test datadir
 (`BasicTestingSetup::m_path_root`), to the standard terminal output.
 
+The `TEST_BITCOIN_PATH` environment variable, if set, determines the
+location of the test datadir, which is recreated for each individual
```

Comment:
Useful to specify which tests this affects?

Reply:
Do you mean which executables (as this PR's description does, bench, fuzz)? That's a good idea, I can add this text (or similar) to the READMEs for those too.

Reply:
Yep pretty much what you did in the PR description